### PR TITLE
Add scripts for 3x recall rings & update item_usable table

### PR DESCRIPTION
**Pull request by Snugglesffxi**
Added missing scripts for [Pashhow Ring](https://ffxiclopedia.fandom.com/wiki/Pashhow_Ring), [Jugner Ring](https://ffxiclopedia.fandom.com/wiki/Jugner_Ring), and [Meriphataud Ring](https://ffxiclopedia.fandom.com/wiki/Meriphataud_Ring). Updated the item_usable table with animation ids and times. Rings are now usable :)

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/items/recall_ring_jugner.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+require("scripts/globals/status")
+require("scripts/globals/keyitems")
+require("scripts/globals/teleports")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:hasKeyItem(xi.ki.JUGNER_GATE_CRYSTAL) == false then
+       result = 445
```
Comment (claywar) [inline scripts/globals/items/recall_ring_jugner.lua:15]: This (and similar) conditionals could be simplified to `if not target:hasKeyItem(xi.ki.JUGNER_GATE_CRYSTAL) then`

File: scripts/globals/items/recall_ring_jugner.lua [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    end
+    return result
+end
+
+item_object.onItemUse = function(target)
+    target:addStatusEffectEx(xi.effect.TELEPORT, 0, xi.teleport.id.JUGNER, 0, 1)
+end
+
+return item_object
```
Comment (claywar) [inline scripts/globals/items/recall_ring_jugner.lua]: Could you please add a newline to the end of this file, and others that are also missing it?

**Review by claywar — COMMENTED** [review]
Thanks for the cleanup!  Left a couple of minor notes.

File: scripts/globals/items/recall_ring_jugner.lua (hunk 1 above)
Comment (Snugglesffxi) [reply scripts/globals/items/recall_ring_jugner.lua:15]: You got it boss! :)

**Review by TeoTwawki — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Pull request merged**